src/main/java/com/commercetools/service/paypalPlus/PaypalPlusPaymentService.java

```diff
@@ -25,4 +27,27 @@
     CompletionStage<Payment> execute(@Nonnull Payment payment, @Nonnull PaymentExecution paymentExecution);
 
     CompletionStage<Payment> lookUp(@Nonnull String paymentId);
```

Comment:
what the name _lookUp_? why not widely used `get` or `getById`, or `find`, or `findById`, like we use in all SphereServices

Reply:
It's a name from Paypal Plus

Reply:
i see `Payment#get(APIContext, String)`, i see HTTP GET/POST requests, but i've never seen _lookUp_ as a method or endpoint name.

(I see _Look Up Payment_ as a chapter name in the documentation, but don't think it is a good reference for a method name :) )

Pls, use more conventional programmist-friendly names

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/57/commits/fcabee91462a48d3cf06f020fae4404e360d941f